blivet/util.py

```diff
@@ -714,6 +714,17 @@ def compare(first, second):
     else:
         return (first > second) - (first < second)
 
+
+def dedup_list(alist):
```

Comment:
Please change the name to something like ``deduplicate_list(alist)`` or something but ``dedup_list`` isn't really self explanation.
Also please add doc string here.